src/myframe.h

```diff
@@ -925,6 +928,23 @@ struct MyFrame : wxFrame {
                 ShowFullScreen(!IsFullScreen());
                 if (IsFullScreen()) sw->Status(_(L"Press F11 to exit fullscreen mode."));
                 break;
+            case A_ZEN_MODE:
+                if (!IsFullScreen()) {
+                    wxToolBar *wtb = this->GetToolBar();
+                    wxStatusBar *wsb = this->GetStatusBar();
+                    if (wtb != nullptr)

```

Comment:
these are `nullptr` when they're currently not shown? if so how do they turn back on?

Reply:
Depending on the user setting, the toolbar and statusbar remain nullptr. These are initalized on app start:

1. https://github.com/aardappel/treesheets/blob/master/src/myframe.h#L675
2. https://github.com/aardappel/treesheets/blob/master/src/myframe.h#L607

In my understanding - if the user setting defaults to no statusbar, then keep this.
Does this sound right or do I get it wrong?

Reply:
ah ok, then that makes sense